src/contracts/GPv2AllowListAuthentication.sol

```diff
@@ -24,6 +28,10 @@ contract GPv2AllowListAuthentication is GPv2Authentication, StorageAccessible {
         _;
     }
 
+    function setManager(address manager_) external onlyManager {
```

Comment:
As @fedgiac commented, we should make the proxy admin the manager setter. In this case I would resort the the code I pasted in the issue itself.